atomic_reactor/plugins/pre_add_filesystem.py

```diff
@@ -168,6 +169,27 @@ def get_default_image_build_conf(self):
 
         return StringIO(self.DEFAULT_IMAGE_BUILD_CONF.format(**kwargs))
 
+    def update_config_from_dockerfile(self, config):
+        """Updates build config with values from the Dockerfile
+
+        Updates:
+          * set "name" from LABEL com.redhat.component (if exists)
+          * set "version" from LABEL version (if exists)
+          * set "release" from LABEL release (if exists)
+        """
+        labels = Labels(df_parser(self.workflow.builder.df_path).labels)
+        for config_key, label in (
+            ('name', Labels.LABEL_TYPE_COMPONENT),
+            ('version', Labels.LABEL_TYPE_VERSION),
+            ('release', Labels.LABEL_TYPE_RELEASE),  # TODO: does this key exist in config file?
```

Comment:
It may, but it likely won't. Hard coding the release value would require a new commit for every new build. When release value is not defined, the createImage task will query Koji for the next release value that should be used. This is similar to what atomic-reactor does via bump_release plugin.

However! I think we should just let the release value be dynamically determined always. Here's what I'm thinking. If we set `name`, `version`, and `release` for the createImage task, there's going to be conflict when the container build is imported into Koji because it will share the same NVR. Obviously, we don't want that. I think it still makes sense to use the same `name` and `version`. Any of this MUST be overwritten by values from image-build.conf.

Reply:
sounds reasonable, only `name` and `version` then, thanks.

Reply:
updated